### ISSUE TITLE
[Feature request] Insert picture from URL

### ISSUE THREAD

**Issue by Primokorn** [tootsuite/mastodon#1162]
Hi,
It seems there's no opened ticket about that.
It would be great to be able to insert a picture/gif (video maybe) from a custom URL instead of browsing the local files we have on the device.

Thank you.

**Comment by dominix** [comment]
I may suggest that posting url or text would also be possible by url. in fact we need a bookmarklet posting url like there is on facebook, tweeter, diaspora, linkedIn , ...

**Comment by krainboltgreene** [comment]
This is called "Oneboxing" and there are quite a few ways to handle this, not just with images.

**Comment by mike-burns** [comment]
The Paperclip gem (are you using it for this part?) understands URLs natively, as an IO adapter.

Note that this feature exposes the direct IP address of the server.

**Comment by krainboltgreene** [comment]
@mike-burns I think they mean linking to a picture in a post.

**Comment by dominix** [comment]
something like "https://framasphere.org/bookmarklet?url=whatever" so you can share any url, type your comment or push a picture

**Comment by hsn10** [comment]
Yeah, This is needed. What about CTRL-V image from clipboard directly into post. This will be good too. Sites like https://snipboard.io/ can do it.

**Comment by JimmyBoyHalpert** [comment]
Is there any chance of this happening soon? Is there any known workaround that will work with embedding an image in a toot while using  a URL? I want to try making a bot that does media posts that make use of image urls
